src/main/java/org/jabref/logic/importer/SearchBasedFetcher.java

```diff
@@ -17,4 +17,8 @@
      * @return a list of {@link BibEntry}, which are matched by the query (may be empty)
      */
     List<BibEntry> performSearch(String query) throws FetcherException;
+
+    default boolean supportsPaging() {
```

Comment:
It would be nice to have an [ADR](https://adr.github.io/madr/) on why `supportsPaging()` was used and not the way through `interface`s. -- Meaning: `instanceof PagedSearchBasedFetcher` vs. `isSearchBased()`. Does the code feel more clean with `supportsPaging`?

Don't know what [Java by Comparison](https://java.by-comparison.com/) says about that.

Reply:
Its just my personal preference, but I'm fine with both ways.

Reply:
Since Tobias already approved it, I would say: keep as is!